app/commands/v2/unpublish.rb

```diff
@@ -124,34 +124,18 @@ def find_unpublishable_content_item
           allowed_states = %w(draft)
         end
 
-        content_item = ContentItem.where(
-          content_id: content_id,
-          locale: locale,
-          state: allowed_states
-        ).lock.first
+        content_item = document.content_items.where(state: allowed_states)
```

Comment:
We shouldn't need a lock here anymore, I think rather than using allowed_states we'll be able to just lookup `document.draft` or `document.live`